strongbox-commons/src/test/resources/pom.properties

```diff
@@ -0,0 +1,4 @@
+#Created by Apache Maven 3.3.9
```

Comment:
Generally, such test files should be produced by your tests, in order to make sure that things are reproducible and that each test has the resources it needs. Otherwise, when you look at the resources themselves, you have no clue which test owns them. This is a good testing practice to follow.